spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```

Comment:
Take care here! this is from the other pr! For now just delete this file and we are going to review it on the other pr (it doesn't make sense to block this PR because of this, since it's going to have some corrections)